src/clint.py

```diff
@@ -3188,7 +3188,8 @@ def CheckLanguage(filename, clean_lines, linenum, file_extension,
         if match:
             error(filename, linenum, 'runtime/deprecated', 4,
                   'Accessing list_T internals directly is prohibited '
-                  '(hint: see commit d46e37cb4c71)')
+                  '(hint: see'
+                  'https://github.com/neovim/neovim/wiki/List-management-in-Neovim)')
```

Comment:
Maybe the parentheses can be removed, so that the closing parenthesis will not be treated as a part of the link in output.

Reply:
Good thinking. Done.